fe/projects/AspNetCore/src/areas/Orders/components/OrderItemForm/OrderItemForm.js

```diff
@@ -19,7 +19,8 @@ const OrderItemForm = (props) => {
         dispatch({ type: "SET_IS_LOADING", payload: true });
 
         const requestPayload = {
-            id: props.id
+            id: props.id,
```

Comment:
why to touch it, is it connected in any way with product cancellation?

Reply:
Yes, I added a check for the order item status in the API controller, so I needed to retrieve the current status of the order item.